udfs/community/README.md

```diff
@@ -23,6 +23,7 @@ SELECT bqutil.fn.int(1.684)
 * [translate](#translateexpression-string-characters_to_replace-string-characters_to_substitute-string)
 * [url_keys](#url_keysquery-string)
 * [url_param](#url_paramquery-string-p-string)
+* [zero_norm](#zero_norm)
```

Comment:
Since your function name is zeronorm, please keep it consistent.

Reply:
Also, in order for GitHub to properly link it, the ref link should be:
```
#zeronormx-any-type-meanx-float64-stddevx-float64
```